cumulus/polkadot-omni-node/lib/src/common/spec.rs

```diff
@@ -485,3 +500,70 @@ where
 		}
 	}
 }
+
+async fn parachain_informant<Block: BlockT>(
+	relay_chain_interface: Arc<dyn RelayChainInterface>,
+	sync_service: Arc<dyn SyncOracle + Send + Sync>,
+	para_id: ParaId,
+) -> () {
+	// Log backed blocks.
+	let pending_candidates =
+		match pending_candidates(relay_chain_interface.clone(), para_id, sync_service).await {
+			Ok(pending_candidates_stream) => pending_candidates_stream.fuse(),
+			Err(err) => {
+				log::error!("Unable to retrieve pending candidate stream: {err:?}.");
```

Comment:
To add to @skunert comment related to the logs, I think we can use just `sc_tracing::tracing`, instead of the `log` crate. Not sure why we keep both in omni node - I think we should use just `sc_tracing::tracing` also for the existing logs, but not mandatory to be done in this PR. 

The tracing-subscriber that's usually set up when running the node accepts both, but `tracing` based events are structured, and can make it easier to parse logs.

Reply:
Sounds good, thanks!